### PR TITLE
GH-1932 Override default CDN host

### PR DESCRIPTION
**Pull request by GordonSmith**
Add ability to replace viz.hpccsystems.com with an alternative source.

Fixes GH-1932

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jchambers-ln Please review

File: src/loader.js [hunk 1]
```diff
@@ -286,13 +287,18 @@
...
                     port: "",
                     pathname: "/" + version + "/dist-amd"
                 });
             }
 
             return {
+                cdnHost: function(_) {
+                    if (!arguments.length) return cdnHost;
+                    cdnHost = _;
+                    return this;
```
Comment (jbrundage) [inline src/loader.js]: Doesn't *this* = *window* in that context?

**Comment by hamiltwp** [comment]
It appears to be working with current DSP version.

File: src/loader.js (hunk 1 above)
Comment (GordonSmith) [inline src/loader.js]: No "this" === the object being returned:  https://github.com/GordonSmith/Visualization/blob/a82e8d2afe2abe32a073587fd4427c49e8533110/src/loader.js#L296-L352
Its to enable the method chaining as per the unitTest

**Comment by jchambers-ln** [comment]
looks good

**Comment by hamiltwp** [comment]
DDL from DSP looks right.  Filter default in DDL is only table 2

**Pull request merged**